package/base-files/files/sbin/sysupgrade

```diff
@@ -60,7 +60,7 @@ export INSTALLED_PACKAGES=${ETCBACKUP_DIR}/installed_packages.txt
 
 IMAGE="$1"
 
-[ -z "$IMAGE" -a -z "$NEED_IMAGE" -a $CONF_BACKUP_LIST -eq 0 -o $HELP -gt 0 ] && {
+[ -z "$IMAGE" ] && [ -z "$NEED_IMAGE" ] && [ "$CONF_BACKUP_LIST" -eq 0 ] || [ "$HELP" -gt 0 ] && {
```

Comment:
I think you should separate the cosmetic changes (like this one) from the "functional" changes (space in filename) and make separate commits for them.

Reply:
The cosmetic changes were so tittle that I kept it together with the shellcheck fixes. However, should label that commit on the most important aspect (the files with space fix) and not the minor issue (shellchecks).

Anyway, I splitted the commit.

Reply:
I don't think it's a good idea to have strings in integer comparison.